core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -124,6 +124,14 @@ oppia.factory('PlayerTranscriptService', ['$log', function($log) {
       }
       return transcript[index];
     },
+    getLastAnswerOnActiveCard: function(index) {
```

Comment:
Add a Karma test for this; in general, always make sure stuff you add is tested where possible.

Reply:
Done